pkg/server/storage/mysql.go

```diff
@@ -55,6 +54,29 @@ func (my *MysqlStorage) QueryHostkeys() (ResultIterator, error) {
 	return &SqlResultIterator{Rows: rows}, nil
 }
 
+func (my *MysqlStorage) RecordGithubMapping(mapping map[string]string) error {
+	for ssoIdentity, githubUser := range mapping {
```

Comment:
This will execute `len(mapping)` queries. Maybe we want to batch it or sleep in between queries?